docclean/autoencoder.py

```diff
@@ -0,0 +1,105 @@
+import tensorflow as tf
+
+
+class Autoencoder():
+    """
+    Autoencoder class.
+
+    Args:
+
+        img_rows (int): Number of pixel rows
+
+        img_cols (int): Number of pixel columns
+
+        nchans (int): Number of color channels
+
+        mixed_precision (bool): Train with fp16 mixed precision
+
+        early_stopping (int): Patience for early stopping
+
+
+    """
+
+    def __init__(self, img_rows: int = 256, img_cols: int = 256, nchans: int = 3, mixed_precision: bool = False,
+                 early_stopping: int = 25):
+        self.img_rows = img_rows
+        self.img_cols = img_cols
+        self.channels = nchans
+        self.early_stopping = early_stopping
+        self.img_shape = (self.img_rows, self.img_cols, self.channels)
+
+        if mixed_precision:
+
+            self.policy = mixed_precision.Policy('mixed_float16')
```

Comment:
mixed_precision is a `bool`. Do you need to import this library from tensorflow? Alternatively, if you're not using it, you just could remove it

Reply:
Commit 9767677 fixes this.